src/ec/packages/ec-component-text-area/text-area.html.twig

```diff
@@ -0,0 +1,83 @@
+{#
+  Parameters:
+  - "id" (string) (default: '')
+  - "disabled" (boolean) (default: false)
+  - "invalid" (boolean) (default: false)
+  - "name" (string) (default: '')
+  - "hide_label" (boolean) (default: '')
+  - "placeholder" (string) (default: '')
+  - "rows" (int) (default: 4)
+  - "extra_classes" (optional) (string) (default: '') Extra classes (space separated) for the text area
+  - "extra_attributes" (optional) (array) (default: []) Extra attributes for text area
+    - "name" (string) Attribute name, eg. 'data-test'
+    - "value" (string) Attribute value, eg: 'data-test-1'
+  Blocks:
+  - "helper_text"
+  - "invalid_text"
+  - "label"
+#}
+
+{# Internal properties #}
+{% set _id = id|default('') %}
+{% set _disabled = disabled|default(false) %}
+{% set _invalid = invalid|default(false) %}
+{% set _name = name|default('') %}
+{% set _hide_label = hide_label|default(false) %}
+{% set _placeholder = placeholder|default('') %}
+{% set _rows = rows|default(4) %}
+
+{% set _helper_text = helper_text|default('') %}
+{% set _invalid_text = invalid_text|default('') %}
+{% set _label = label|default('') %}
+
+{% set _css_class = ['ecl-text-area'] %}
+{% if _invalid %}
+  {% set _css_class=_css_class|merge(['ecl-text-area--invalid']) %}
```

Comment:
Not a blocker, yet it "hurts" the eyes not having spaces between assignments. Later in the same document, there are spaces. Too pity there is no prettier for that ...